patroni/__init__.py

```diff
@@ -48,13 +48,19 @@ def load_dynamic_configuration(self):
                 logger.warning('Can not get cluster from dcs')
 
     def get_tags(self):
-        return {tag: value for tag, value in self.config.get('tags', {}).items()
+        tags = {tag: value for tag, value in self.config.get('tags', {}).items()
                 if tag not in ('clonefrom', 'nofailover', 'noloadbalance') or value}
+        tags.update(self.ha.get_dynamic_tags())
```

Comment:
`patroni.tags` currently used in `Ha` and `Api`.
Pattern in both cases similar:
```python
# Ha
if self.patroni.tags:
   data['tags'] = self.patroni.tags

# Api
response.update({'tags': patroni.tags} if patroni.tags else {}) 
```

Lets just create method or property in Ha which will mix patroni.tags and dynamic tags and return something like: `{'tags': {'tag1': 'value1'}}` if we have some tags to expose and `{}` if we don't.

Later we can use it as:
```python
foo.update(patroni.ha.get_tags())
```

P.S. it make sense to expose dynamic `nosync` tag via API as well